src/bindings/Rimraf.re

```diff
@@ -0,0 +1,16 @@
+module Rimraf = {
```

Comment:
This module could use a comment (and an mli)

Reply:
Added a comment and signature